buildkit/packaging/archlinux.py

```diff
@@ -13,11 +13,12 @@
 
 from ..common import PACKAGING_DIR, BuildkitAbort, get_logger, get_resources_dir, ensure_empty_dir
 from ._common import (
-    DEFAULT_BUILD_OUTPUT, SHARED_PACKAGING, process_templates)
+    SHARED_PACKAGING, process_templates)
 
 # Private definitions
 
 # PKGBUILD constants
+ARCHLINUX_BUILD_OUTPUT = Path('out/Release')
```

Comment:
Why is the build output path hardcoded here? Is there something wrong with `out/Default` or some other user-specified build output directory?

Reply:
I had a very annoying issue using `out/Default` that I was not able to solve, so I thought the default gn path could be ok. But since, to be sure, I now tried again to reproduce the issue, and everything worked well (clearly something else was causing it) I will revert back using you default path.

I'm sorry, as always rushing thinks makes you make mistakes.